render/data/data.go

```diff
@@ -54,7 +54,10 @@ func (d *Data) GetAggregation(id uint32) (string, error) {
 	if err != nil {
 		return function, err
 	}
-	if function == "any" || function == "anyLast" {
+	switch function {
+	case "any":
+		return "first", nil
+	case "anyLast":
 		return "last", nil
 	}
 	return function, nil
```

Comment:
Nit blocking. May be move to default switch section ?

Reply:
Done in [4fd0fdd](https://github.com/go-graphite/graphite-clickhouse/pull/281/commits/4fd0fddc52c05739453e15c98f31994877917ab0)